haiku-games/bong/bong-1.4~git.recipe

```diff
@@ -0,0 +1,55 @@
+SUMMARY="BeOS version of Pong"
+DESCRIPTION="Bong is a FreeWare game for BeOS,based on the old arcade classic Pong"
+HOMEPAGE="https://github.com/HaikuArchives/Bong"
+COPYRIGHT="1997 by Gertjan van Ratingen"
+LICENSE="bong"
+REVISION="1"
+srcGitRev=906d822b79bdae9328fe743dffe2871f96a30e0e
+SOURCE_URI="$HOMEPAGE/archive/$srcGitRev.tar.gz"
+SOURCE_FILENAME="bong-1.4~$srcGitRev.tar.gz"
+SOURCE_DIR="Bong-$srcGitRev"
+
+PATCHES="bong-$portVersion.patchset"
+
+ARCHITECTURES="?x86_gcc2 ?x86 x86_64"
+
+PROVIDES="
+	bong = $portVersion
+	app:bong = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	makefile_engine
+	cmd:gcc
+	cmd:make
+	"
+
+GLOBAL_WRITABLE_FILES="
+	settings/bong auto-merge
+	"
+
+USER_SETTINGS_FILES="
+	settings/bong template data/bong/defaultconfig
+	"
+
+BUILD()
+{
+	make $jobArgs OBJ_DIR=objects MACHINE=BePC CC=g++ LD=g++
+	mkdir -p settings
+	echo "my bong prefs" >settings/bong
```

Comment:
The "my bong prefs" was just a way to create a file that could serve as a settings file. But it was just an example.

You noticed in https://github.com/HaikuArchives/Bong/blob/master/README.txt#L48 that Bong creates a file in /boot/home/config/settings/Bong_prefs.
Since this is not a system setting (i.e. in  /boot/system/settings/), but a user setting, this does not need to have a GLOBAL_WRITABLE_FILES nor a USER_SETTINGS_FILES.

So you can:
* completely drop GLOBAL_WRITABLE_FILES and USER_SETTINGS_FILES.
* remove everything in BUILD after the the call to make.
* change INSTALL to:
```
INSTALL()
{
	install -d $appsDir
	install -t $appsDir objects/bong
	addAppDeskbarSymlink $appsDir/bong Bong
}
```